.travis.yml

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#
+# Tencent is pleased to support the open source community by making TubeMQ available.
+#
+# Copyright (C) 2012-2019 Tencent. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at
+#
+# https://opensource.org/licenses/Apache-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+sudo: required
+dist: trusty
+language: java
+
+jdk:
+  - oraclejdk8
+
+cache:
+  directories:
+    - $HOME/.m2
```

Comment:
Maybe we should exclude our own artifacts by

```
before_cache:
  - rm -rf $HOME/.m2/repository/com/tencent/tubemq/
```

Reply:
Currently, we just `mvn verify`. But I think it's fine to add this line.